src/component/data.js

```diff
@@ -0,0 +1,8 @@
+var Data = {
```

Comment:
I was thinking about how to store data in memory. And perhaps in WebSQL or LocalStorage. This was a lame attempt at structuring that.